site/docs/encoding/type.md

```diff
@@ -3,25 +3,32 @@ layout: docs
 title: Type
 permalink: /docs/type.html
 ---
-<!-- TODO: Intro for type -->
 
+If a field is specified, the channel definition **must** describe the encoded data's [type of measurement (level of measurement)](https://en.wikipedia.org/wiki/Level_of_measurement).
+The supported data types are: `"quantitative"`, `"temporal"`, `"ordinal"`, and `"nominal"`.
 
-### Data Type
+## Quantitative
+Quantitative data expresses some kind of quantity. Typically this is numerical data. For example `7.3`, `42.0`, `12.1`.
 
-If a field is specified, the channel definition **must** describe the encoded data's [type of measurement (level of measurement)](https://en.wikipedia.org/wiki/Level_of_measurement).
-The supported data types are:
+## Temporal
+Temporal data supports date-times and times. For example `2015-03-07 12:32:17`, `17:01`, `2015-03-16`.
+
+Note that when a `"temporal"` type is used for a field, Vega-Lite will treat it as a continuous field and thus will use a [`time` scale](scale.html#time) to map its data to visual values. For example, the following bar chart shows the dimension of the bars of a time scale.
+
+<span class="vl-example" data-name="bar_month_temporal"></span>
+
+### Casting a Temporal Field as an Ordinal Field
```

Comment:
Casting is a bad term for this. Casting means to convert types but in our case we are treating data differently. I'd prefer to say it that way.

Reply:
I disagree.  When I talk to Jeff, he also that this casting seems natural.

Reply:
Okay